packages/integration-tests/tests/__snapshots__/flat-config-types.test.ts.snap

```diff
@@ -16,7 +16,7 @@ exports[`flat-config-types eslint should work successfully 1`] = `
         "line": 38,
         "message": "'_otherCases' is defined but never used.",
         "messageId": "unusedVar",
-        "nodeType": "Identifier",
+        "nodeType": null,
```

Comment:
I ... don't know why this is happening. But the actual rule seems to work internally.

Reply:
https://github.com/typescript-eslint/typescript-eslint/commit/812787313626c927c506c58e00aff3857f2c163c / #8640

This commit changed the `context.report` from `{ node, ... }` to `{ loc, ... }`
Hence the `nodeType` is no longer visible in the output as the rule is not reporting against the node.